application/src/main/java/org/thingsboard/server/controller/UserController.java

```diff
@@ -151,9 +153,10 @@ public User getUserById(
             processDashboardIdFromAdditionalInfo(additionalInfo, DEFAULT_DASHBOARD);
             processDashboardIdFromAdditionalInfo(additionalInfo, HOME_DASHBOARD);
             UserCredentials userCredentials = userService.findUserCredentialsByUserId(user.getTenantId(), user.getId());
-            if (userCredentials.isEnabled() && !additionalInfo.has("userCredentialsEnabled")) {
-                additionalInfo.put("userCredentialsEnabled", true);
+            if (userCredentials.isEnabled() && !additionalInfo.has(USER_CREDENTIALS_ENABLED)) {
```

Comment:
I believe this parameter is also a subject of removal from additional info. Maybe store it as a separate DB field? PR seems incomplete without this change.